splunk-to-segment/splunk-to-segment.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"encoding/json"
+	"os"
+	"os/exec"
+)
+
+type ExpectedOutput struct {
+	MessageID  string                 `json:"messageId"`
+	Timestamp  string                 `json:"timestamp"`
+	Namespace  string                 `json:"namespace"`
+	Type       string                 `json:"type"`
+	UserID     int64                  `json:"userId"`
+	Event      string                 `json:"event"`
+	Properties map[string]interface{} `json:"properties"`
+	Context    map[string]interface{} `json:"context"`
+}
+
+func runScriptFromFile(filePath, scriptPath string) ([]byte, error) {
```

Comment:
Is there a need for an extra layer (go file) for `splunk-to-segment`? those methods seem to be helpers for running scripts, I don't think they should be specific to splunk-to-segment (especially as the filePath and scriptPath can be dynamic)